styles.css

```diff
@@ -0,0 +1,483 @@
+body {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  background-image: url(./images/first_page/img.jpg);
+}
```

Comment:
- [ ] You're currently using the background image for  the whole body on both the index and the about page, but as you can see from the template, only the main section uses that background, please make sure to change it.

Reply:
I could check that both uses the background image, however just for the initial area.